src/styles/global.css

```diff
@@ -18,11 +18,17 @@ body {
   font-style: normal;
 }
 
-p, a {
+p, a, button, ul, li, dt, dl {
```

Comment:
Maybe better to just add all these styles to body and then override headings so we have a default style for all text as this list will just grow and grow as we need it